config/hancestro.yml

```diff
@@ -4,16 +4,17 @@ idspace: HANCESTRO
 base_url: /obo/hancestro
 
 products:
-- hancestro.owl: https://raw.githubusercontent.com/EBISPOT/ancestro/master/hancestro_inferred.owl
+- hancestro.owl: https://raw.githubusercontent.com/EBISPOT/hancestro/main/hancestro.owl
 
 term_browser: ontobee
 example_terms:
 - HANCESTRO_0004
 
 entries:
 - exact: /hancestro.owl
-  replacement: https://raw.githubusercontent.com/EBISPOT/ancestro/master/hancestro.owl
-- exact: /hancestro_inferred.owl
-  replacement: https://raw.githubusercontent.com/EBISPOT/ancestro/master/hancestro_inferred.owl
-- exact: /hancestro_bfo.owl
-  replacement: https://raw.githubusercontent.com/EBISPOT/ancestro/master/hancestro_bfo.owl
```

Comment:
See above - can you redirect this to some file, perhaps an empty one with a comment? Does hancestro_bfo.owl not exist anymore?

Reply:
The file hancestro_bfo.owl doesn't exist anymore but the artefact does - it's now called hancestro-full.owl and hancestro.owl